tests/features/contact_form.feature

```diff
@@ -12,8 +12,6 @@ Feature: Contact Form
       | contact_form |
     When I am on "contact"
     Then I should see "Contact - European Commission" in the "title" tag
-    When I fill in "Your name" with "Chuck"
```

Comment:
The test is not correct anymore because an anonymous user **can** insert a name and an e-mail in the contact form. 
So, we must reinsert these 2 lines